src/main/java/org/opensearch/security/dlic/rest/validation/AbstractConfigurationValidator.java

```diff
@@ -297,4 +314,16 @@ public String getMessage() {
     protected final boolean hasParams() {
         return param != null && param.length > 0;
     }
-}
\ No newline at end of file
+
+    private boolean hasNullElement(JsonNode node) {
+        for (JsonNode jsonNode: node) {
+            if(jsonNode.isNull() && node.isArray()) {
```

Comment:
If `jsonNode` is `null` and `node` is not an array, it is not necessary to call `hasNullElement`. We need to call `hasNullElement()` only if `jsonNode` is object or an array:
```
if (jsonNode.isNull()) {
  if (node.isArray()) {
    return true;
  }
} else if (jsonNode.isContainerNode()) {
  if (hasNullElement(jsonNode)) {
    return true;
  }
}
```

Reply:
fixed